migrations/1753214286445_txs-status-index.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable camelcase */
+
+exports.shorthands = undefined;
+
+exports.up = pgm => {
+  pgm.createIndex('txs', 'status', {
+    where: 'canonical = TRUE AND microblock_canonical = TRUE',
+    name: 'idx_txs_status_optimized'
```

Comment:
No need to provide a name, `node-pg-migrate` will define one for you with the current standards we use